save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -46,4 +51,35 @@ class VulnerabilityService(
         val description = vulnerabilityDescriptionRepository.findByVulnerabilityName(name)?.description.orEmpty()
         return vulnerabilityRepository.findByName(name)?.toDtoWithDescription(description) ?: VulnerabilityDto.empty
     }
+
+    /**
+     * @param vulnerabilityDto dto of new vulnerability
+     */
+    @Transactional
+    fun save(vulnerabilityDto: VulnerabilityDto) {
+        val number = (vulnerabilityRepository.findMaxId() ?: 0) + 1
```

Comment:
do we need to assign the id then?
or maybe save in db with empty\stub name and then update it using ID assigned by database?

Reply:
We need a unique name. But it may not match the id in the database.

Reply:
I guess there is a contain that name is unique and we will get an error in case of two requests at same time

Reply:
Okay, I agree. I'll change that in the next PR.